### PR TITLE
Conformance configuration tests, part 1

### PR DESCRIPTION
**Pull request by xpivarc**
### What this PR does
Cover part of configuration tests in conformance testing. See https://github.com/kubevirt/kubevirt/blob/main/docs/conformance.md

Fixes #

### Release note

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go [hunk 1]
```diff
@@ -2095,11 +2098,68 @@ var _ = Describe("Validating VMICreate Admitter", func() {
...
+			Expect(err).To(Not(HaveOccurred()))
+
+			ar := &admissionv1.AdmissionReview{
+				Request: &admissionv1.AdmissionRequest{
+					Resource: webhooks.VirtualMachineInstanceGroupVersionResource,
+					Object: runtime.RawExtension{
+						Raw: vmiBytes,
+					},
+				},
+			}
```
Comment (fossedihelm) [inline pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go:2111]: +1

**Review by fossedihelm — COMMENTED** [review]
Just a wrong header

**Comment by fossedihelm** [comment]
@xpivarc failure is relevant, I think

**Comment by xpivarc** [comment]
> @xpivarc failure is relevant, I think

Fixed, typo `Kubevirt` -> `KubeVirt`

**Review by fossedihelm — APPROVED** [review]
Thank you!

**Pull request merged**